initializr-generator/src/main/java/io/spring/initializr/generator/buildsystem/maven/MavenBuild.java

```diff
@@ -134,4 +138,66 @@ public String getPackaging() {
 		return this.packaging;
 	}
 
+	public void resource(String directory, Consumer<ResourceBuilder> customizer) {
+		customizer.accept(this.resources.computeIfAbsent(directory, (key) -> new ResourceBuilder(directory)));
+	}
+
+	public List<Resource> getResources() {
+		return this.resources.values().stream().map(ResourceBuilder::build).collect(Collectors.toList());
+	}
+
+	/**
+	 *
+	 * Builder to create a {@link Resource}.
+	 *
+	 *
+	 */
+	public static final class ResourceBuilder {
+
+		private String directory;
+
+		private List<String> includes = new ArrayList<>();
```

Comment:
A maven resource has more than a directory and includes, see [the POM reference](https://maven.apache.org/pom.html#Resources).

Reply:
Thanks! I'll make the needed changes as soon as possible!